TKeazirian.HTTPServer/Utils/FileUtility.cs

```diff
@@ -0,0 +1,23 @@
+namespace TKeazirian.HTTPServer.Utils;
+
+public static class FileUtility
+{
+    public static string GetPath()
+    {
+        string path;
+        var currentDirectory = Directory.GetCurrentDirectory();
+        if (currentDirectory.Contains("/bin/Debug"))
```

Comment:
@Aoxian how about this fix? for [this](https://github.com/t-keazirian/HTTPServer/pull/35/files/fbe7713c14aecb9f032bbd7fa1b43ddde3d0055c#r927125932) comment.

Reply:
if it works, this looks a bit better. though it still appears to be an absolute path, its not specific to your mac.

Reply:
It would be an interesting exercise to clone the repo on different machines to see if there are issues.  Like, what happens when you try to build and test on a Windows machine?

Reply:
> though it still appears to be an absolute path

I guess due to the `Contains` it doesn't have to be absolute.

Reply:
Yeah, I think the `Contains` makes it so that I can just specify the `bin/Debug` and that is (I believe) the way that Rider sets up all C# folders/projects when you Run it.

Reply:
Have you considered, instead of making the code repo conform to Rider; instead make sure everything works with the `dotnet` command, and then change the Rider settings to conform to it?

Reply:
I am pretty sure there are configurations for building, testing and running software in Jetbrains products.

Reply:
the more I understand the issue, the more it sounds like you are programing to make your IDE happy here.

Reply:
Yes, I agree - Marie and I looked into it pretty extensively today.  We could not figure out a way to get the Rider settings to conform to "not run" from that folder...I could make it so that everything works with `dotnet run` command and just never use the `Run` button in Rider, if that's the case...but we did not figure out a way to configure the Rider settings to fix it.

Reply:
Seth used a `PUBLIC_DIR` global variable to get around it in Java [here](https://github.com/sethpthomas91/http-server/blob/ffaeac30478bcd08653af4f825f6b263e905214a/.github/workflows/combinedWorkflow.yml), but I'm not sure if it would work the same way for C#.

Reply:
This is a good conversation, but I do not want to hold up progress any more. If this was a client team we were on, I would hold my ground though, and suggest not changing code around tools that not everyone on the team may be using (Rider in this case).

Reply:
I fixed it!!!!